certora/applyMetaMorphoMunging.patch

```diff
@@ -0,0 +1,106 @@
+diff -ruN MetaMorpho.sol MetaMorpho.sol
+--- MetaMorpho.sol
++++ MetaMorpho.sol
+@@ -19,7 +19,6 @@
+ import {UtilsLib} from "../lib/morpho-blue/src/libraries/UtilsLib.sol";
+ import {SafeCast} from "../lib/openzeppelin-contracts/contracts/utils/math/SafeCast.sol";
+ import {SharesMathLib} from "../lib/morpho-blue/src/libraries/SharesMathLib.sol";
+-import {MorphoLib} from "../lib/morpho-blue/src/libraries/periphery/MorphoLib.sol";
+ import {MarketParamsLib} from "../lib/morpho-blue/src/libraries/MarketParamsLib.sol";
+ import {IERC20Metadata} from "../lib/openzeppelin-contracts/contracts/token/ERC20/extensions/IERC20Metadata.sol";
+ import {MorphoBalancesLib} from "../lib/morpho-blue/src/libraries/periphery/MorphoBalancesLib.sol";
+@@ -45,9 +44,7 @@
+     using UtilsLib for uint256;
+     using SafeCast for uint256;
+     using SafeERC20 for IERC20;
+-    using MorphoLib for IMorpho;
+     using SharesMathLib for uint256;
+-    using MorphoBalancesLib for IMorpho;
```

Comment:
We can probably reintroduce that one to minimize the diff. Note that we still would need to avoid to call `expectedSupplyAssets` as the goal of the munging here is to avoid calling `extSloads`